bigtable/helloworld/main.go

```diff
@@ -86,14 +91,18 @@ func main() {
 			log.Fatalf("Could not create column family %s: %v", columnFamilyName, err)
 		}
 	}
+	// [END bigtable_hw_create_table]
 
 	// Set up Bigtable data operations client.
 	// NewClient uses Application Default Credentials to authenticate.
+	// [START bigtable_hw_connect_data]
```

Comment:
Do you not want the preceding comments? Seems useful!

Reply:
The docs already have comments about it, so it seems easier for them to be maintained there. Same for below. Thoughts? 

https://cloud.google.com/bigtable/docs/samples-go-hello

Reply:
SGTM!